packages/core/src/utils.ts

```diff
@@ -199,13 +199,19 @@ export const getGasPriceOverrides = async (
     case 1442:
     case 1101:
       return overridden
-    // On linea and its testnet, just override the gasPrice
+    // On linea testnet, 10x the gas price because of this bug:
+    // https://github.com/ethers-io/ethers.js/issues/4533
     case 59140:
+      if (gasPrice !== null) {
```

Comment:
Why do we only override the `gasPrice` when it's not `null`? Some docs would be nice

Reply:
It just prevents a `gasPrice` value passed in by the caller from being overridden if the `gasPrice` returned by `getFeeData` is `null`. Honestly, this probably isn't necessary given that we generally never pass in our own values for `gasPrice`, `maxFeePerGas`, or `maxPriorityFeePerGas`. Even if we did, I'm not sure I see the use case for including `gasPrice` on a network where it isn't supported. I've mainly continued to do this b/c it was the original pattern implemented by Kelvin for this function. 

If we want to remove this, I would do it for the whole function.

Reply:
I also made a ticket for this.

Reply:
Sounds good